docker/daemon/daemon_src.go

```diff
@@ -34,8 +33,8 @@ type layerInfo struct {
 // (We could, perhaps, expect an exact sequence, assume that the first plaintext file
 // is the config, and that the following len(RootFS) files are the layers, but that feels
 // way too brittle.)
-func newImageSource(ctx *types.SystemContext, ref daemonReference) (types.ImageSource, error) {
-	c, err := client.NewClient(client.DefaultDockerHost, "1.22", nil, nil) // FIXME: overridable host
+func newImageSource(systemCtx *types.SystemContext, ref daemonReference) (types.ImageSource, error) {
+	c, err := newDockerClient(systemCtx)
```

Comment:
Why the rename from `ctx` to `systemCtx`? `ctx` is used throughout the library.

Reply:
to align daemon_dest and daemon_src. But if the guideline is to use ctx, I am fine with that as well.